docs/en/observability/synthetics-run-test.asciidoc

```diff
@@ -0,0 +1,146 @@
+
+Inline monitors are quick and easy to set up.
+To create one, you can copy a journey's content, paste it into Kibana or your {heartbeat} configuration, and save.
+Variables like `page` and `params` will be part of your script's scope.
+However, this type of monitor does not allow you to import third-party packages.
+
+Inline monitors can also be more difficult to manage because each inline monitor can contain only _one_ journey, which must be maintained from within Kibana.
+If you depend on external packages, have your journeys next to your code repository, or wish to embed and manage more than one journey from a single monitor configuration <<synthetics-test-suite,use a test suite>>.
+
+[discrete]
+[[synthetics-inline-journey-local]]
+=== Test locally
+
+To test the inline monitor's journey locally, change into your test's directory
+and pipe the journey's contents to the `npx @elastic/synthetics` command.
+
+Assume, for example, that your inline monitor contains the following code:
+
+[source,js]
+----
+step('load homepage', async () => {
+    await page.goto('https://www.elastic.co');
+});
+step('hover over products menu', async () => {
+    await page.hover('css=[data-nav-item=products]');
+});
+----
+
+To run that journey locally, you can save that code to a file and pipe the file's contents into `@elastic-synthetics`:
+
+[source,sh]
+----
+cat path/to/sample.js | npx @elastic/synthetics --inline
+----
+
+And you'll get a response like the following:
+
+[source,sh]
+----
+Journey: inline
+   ✓  Step: 'load homepage' succeeded (1831 ms)
+   ✓  Step: 'hover over products menu' succeeded (97 ms)
+
+ 2 passed (2511 ms)
+----
+
+[discrete]
+[[synthetics-inline-journey-send]]
+=== Send to {stack}
+
+After testing locally, you can copy the script into your Elastic Synthetics integration or `heartbeat.yml`
+depending on <<uptime-set-up-choose,the approach you chose>>.
+
+--
+include::tab-widgets/uptime-monitoring/run-test-inline-widget.asciidoc[]
+--
+
+[discrete]
+[[synthetics-test-suite]]
+== Use a test suite
+
+If you have a suite of tests, you can use the `@elastic/synthetics` package as a library.
+
+To test an test suite locally, start by writing your tests:
+
+. Create a new https://docs.npmjs.com/cli/v7/commands/npm-init[NPM/Node.js project].
+. Create a `javascript` or `typescript` file that imports your tests.
+All synthetic test files must use the `.journey.ts` or `.journey.js` file extension.
+. Make sure any dependencies are defined with an `import` outside of the `journey` object.
+. Compile everything together.
+
+To see an example of a test suite, see the https://github.com/elastic/synthetics[elastic/synthetics] repository.
+You can use the `todo` directory if you'd like to test an example locally.
+Clone the repository, change into the directory containing the synthetics test suite,
+and install the necessary dependencies including `@elastic/synthetics`:
+
+[source,sh]
+----
+# Check out the synthetics repo and included examples
```

Comment:
Let's stop recommending that repo and instead point folks at https://github.com/elastic/synthetics-demo , I thought I'd removed all references but I must have made an error. The problem with checking out the source repo is it can lead to weird node import errors. Also, we don't update the examples there as frequently